lib/kubernetes-deploy/kubernetes_resource/pod.rb

```diff
@@ -101,6 +101,16 @@ def print_debug_logs?
       exists? && !@stream_logs # don't print them a second time
     end
 
+    def ready?
```

Comment:
I had to pull this out of `private`, so I guess i'm the first one to have a valid reason to call it. I think this is fine as a public method.